Kickstarter-iOS/Views/PledgeViewCTAContainerView.swift

```diff
@@ -109,6 +110,13 @@ final class PledgeViewCTAContainerView: UIView {
         guard let self = self else { return }
         self.delegate?.applePayButtonTapped()
       }
+
+    self.viewModel.outputs.notifyDelegateOpenHelpType
+      .observeForControllerAction()
```

Comment:
This can be `observeForUI()` but doesn't matter too much, just that this doesn't directly present the controller whereas the other one would specifically place the presentation on that queue. The fact that we even have these two different `observeFor` functions was to workaround a bug that might no longer exist so we should dig into that a bit more.